tests/test_integration.py

```diff
@@ -139,6 +139,12 @@ def test_parse_tree(self):
             """)
         )
 
+    def test_to_tree(self):
+        toklists = parse(data)
+        tree = parse_tree(data)
+        self.assertEqual([toklist.to_tree() for toklist in toklists], tree)
+
+
```

Comment:
Since this tests a method in models.py, I'd like the test to be in test_models, and just test the to_tree.